Makefile

```diff
@@ -43,12 +43,10 @@ depend:
 depend-update:
 	dep ensure -update
 
-bin:
-	@mkdir $@
-
 .PHONY: build
-build: | bin ## build binary
```

Comment:
why delete `bin` directory?

Reply:
It gets created automatically by `go build -o bin/...`.